network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -27,10 +44,25 @@ <h1 class="h1-heading">{{ page.title }}</h1>
         <div class="share-button-group-wrapper mt-5" data-share-text="{{page.title}} by @mozilla" data-link="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}"></div>
       </div>
     </div>
+
+    {% if coralTalkServerUrl and page.feature_comments %}
+      <div id="coral-talk-stream"></div>
+    {% endif %}
+
     {% include "./fragments/post_tags.html" %}
   </div>
 {% endblock %}
 
 {% block prefooter %}
   {% include "./fragments/related_posts.html" %}
 {% endblock %}
+
+{% block extra_scripts %}
+  {% if coralTalkServerUrl and page.feature_comments %}
+    <script src="{{ coralTalkServerUrl }}static/embed.js" async onload="
```

Comment:
```suggestion
    <script src="{{ coral_talk_server_url }}static/embed.js" async onload="
```

Reply:
Rather than `async` this should probably be `defer`? (e.g. implicit async load, but don't run only once the DOM the has been parser, rather than when the file finishes transfering)

Reply:
I'm almost certain that `onload="..."` falls foul of CSP (violating inline-js rules), but even if it doesn't, let's not put JS inside an HTML attribute, even if that means a tiny three line `.js` file that also gets loaded with `defer`.

Reply:
Oh, this script is directly from the coral talk admin config...not sure if we should be changing that, tbh. This is also how it's implemented in pni.

Reply:
I see. if this is according to their docs then I guess that's what we'll use for now.